SampleProject/VOKCoreDataManagerTests/DateFormatterTests.m

```diff
@@ -0,0 +1,27 @@
+//
+//  DateFormatterTests.m
+//  SampleProject
+//
+//  Created by Carl Hill-Popper on 4/27/16.
+//  Copyright © 2016 Vokal.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <VOKManagedObjectMap.h>
+
+@interface DateFormatterTests : XCTestCase
+
+@end
+
+@implementation DateFormatterTests
+
+- (void)testDateFormattersLocaleIsPOSIX
+{
+    XCTAssertEqualObjects([VOKManagedObjectMap vok_defaultDateFormatter].locale.localeIdentifier,
+                          VOKDefaultDateFormatterLocaleIdentifier);
+    XCTAssertEqualObjects([VOKManagedObjectMap vok_dateFormatterWithoutMicroseconds].locale.localeIdentifier,
+                          VOKDefaultDateFormatterLocaleIdentifier);
+}
```

Comment:
Unless we move this test to a new test target and add a scheme specifically for it. I haven't done any testing to figure out what region or locale we could use for this that would change the calendar. You can switch it in Settings > General > Language & Region > Calendar.